optaweb-vehicle-routing-backend/README.adoc

```diff
@@ -59,3 +73,39 @@ Services are injected into plugins, which moves the burden of their
 construction and dependency resolution to the IoC container.
 Secondly, plugins implement services' output boundary interfaces to handle
 use case results, e.g. persisting changes to database, sending response to web UI.
+
+== Development Guide
+
+=== Automatic Restart
+
+https://docs.spring.io/spring-boot/docs/2.1.1.RELEASE/reference/htmlsingle/#using-boot-devtools-restart[Automatic restart]
```

Comment:
Any way we can replace that "2.1.1.RELEASE" with "latest" or "LATEST" or "latestFinal" and it still works?
Our docs have such a symlink, I would expect theirs to have that too.

Reply:
Yes, they use `current`. I'll change it.

Ideal solution would be keeping the version synchronized with our Spring Boot dependency version and verifying the link doesn't break with version upgrade during CI build.

I have filed #49.